DQ_IL2_index_raw_msg.py

```diff
@@ -241,20 +248,45 @@ def main(argv):
       ATTEMPT_NO+=1
 
     ##############################################################################################################################
-    # Move files to Local archive folder
+    # Move files to WFLS
```

Comment:
This should be "Moves files to local folder" or something similar

Reply:
Latest commit should address this.